Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -57,21 +33,8 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'com.google.android.gms:play-services-base:17.6.0' // Packages/com.wildlifestudios.wls-ads/Editor/WLSAdsDependencies.xml:18
 // Android Resolver Dependencies Start
-    implementation 'com.scalemonk.libs:ads:1.0.23' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:4
-    implementation 'com.scalemonk.libs:ads-adcolony:4.4.1.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:5
-    implementation 'com.scalemonk.libs:ads-admob:19.7.0.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:6
-    implementation 'com.scalemonk.libs:ads-applovin:9.15.3.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:7
-    implementation 'com.scalemonk.libs:ads-chartboost:8.1.0.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:8
-    implementation 'com.scalemonk.libs:ads-facebook:6.2.1.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:9
-    implementation 'com.scalemonk.libs:ads-fyber:7.6.0.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:10
-    implementation 'com.scalemonk.libs:ads-inmobi:9.0.9.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:11
-    implementation 'com.scalemonk.libs:ads-ironsource:7.1.0.2.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:12
-    implementation 'com.scalemonk.libs:ads-mintegral:15.4.51.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:13
-    implementation 'com.scalemonk.libs:ads-mopub:5.14.0.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:14
-    implementation 'com.scalemonk.libs:ads-smaato:21.5.3.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:15
-    implementation 'com.scalemonk.libs:ads-tiktok:3.1.0.1.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:16
-    implementation 'com.scalemonk.libs:ads-unityads:3.6.0.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:17
-    implementation 'com.scalemonk.libs:ads-vungle:6.8.0.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:18
+    implementation 'com.scalemonk.libs:ads:4.0.0-alpha.1' // Assets/ScaleMonk Ads/Editor/ScaleMonkAdsDependencies.xml:4
```

Comment:
This part is waiting for the release, to place the "to publish" versions instead of beta(s).

Reply:
Sure, @nravasi as mentioned above, that was never intended to happen. This PR only gets merged with final versions.